android/build.gradle

```diff
@@ -39,7 +39,7 @@ ext {
     clientmanagerLibVersionCode = 1
     clientmanagerLibVersionName = "\"1.0.0\""
 
-    serverBaseURL = "\"https://api-internal.qa-platform.mosip.net/\""
+    serverBaseURL = "\"https://api-internal.qa-trinity.mosip.net/\""
```

Comment:
can you pull the latest code, it should be qa-platform

Reply:
Changed to qa-platform